packages/compass-schema-validation/src/components/sample-documents/sample-documents.jsx

```diff
@@ -9,14 +9,38 @@ import {
   cx,
   spacing,
   palette,
+  InlineDefinition,
 } from '@mongodb-js/compass-components';
+import {
+  DOCUMENT_LOADING_STATES as DOCUMENT_LOADING_STATES_MAP,
+  SAMPLE_SIZE,
+} from '../../modules/sample-documents';
+
+const SAMPLE_DEFINITION = [
+  'A sample is fetched from a sample-space of',
+  SAMPLE_SIZE,
+  'randomly selected documents',
+].join(' ');
+
+const DOCUMENT_LOADING_STATES = Object.values(DOCUMENT_LOADING_STATES_MAP);
 
 /**
  * The Sample Documents editor component.
  */
 
-const sampleDocumentsStyles = css({
+const sampleDocumentsSectionStyles = css({
   marginTop: spacing[4],
+  display: 'flex',
+  flexDirection: 'column',
+  gap: spacing[2],
+});
+
+const sampleDocumentsHeaderStyles = css({
+  fontSize: 16,
+  fontWeight: 'bold',
+});
```

Comment:
Leafygreeen has heading components, any particular reason we are using a custom one here?

Reply:
As far as I remember, in Figma this header was of `16px` in size and bold (`bolder` than the weight supported in leafygreen typography) which is why I went ahead with custom definitions.

For now I will switch to using `Subtitle` which is of `18px` in size and will confirm with Claudia later.

Reply:
Addressed by using `Subtitle` from leafygreen